### PR TITLE
Spring 2019 Fanart Voting newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
👀 🎵 👀 

---

The voting is now open for the [Spring 2019 Fanart Contest!](https://osu.ppy.sh/community/contests/78) Get in and place your **15** votes today, the polls will only be open until the beginning of June!

### CODE REVIEW

File: news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Spring 2019 Fanart Contest Voting Now Open!"
+date: 2019-05-25 20:00:00 +0000
+---
+
+With over 130 superbly talented entries to pick from the Spring 2019 Fanart Contest, talent really IS in season. Head on over and cast your 15 votes now - but be quick about it, the votes are only open for **7 days!**
```
[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: pick from in is just straight up wrong here i think but i'll see about rewording the talented leadin thing

File: news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+[![](https://assets.ppy.sh/contests/78/header.jpg)](https://osu.ppy.sh/community/contests/78)
```
[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: no harm in leaving it in and besides, the backing banner is still supposed to link, it just doesn't >_>

File: news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cl8n) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: "pick" isn't the verb of this sentence, you're describing what the entries are for with "to pick from" so u can read as "entries ~~to pick from~~ **in** the Spring 2019 [...]"

either you meant something else (so reword) or you need the "in"

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: i ended up applying this in the reword anyway so nbd

[the remark this answers is not in the document]
Comment (cl8n) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: em dash overlords are not pleased with your change 💢

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: ok i applied this one too

[the remark this answers is not in the document]
Comment (cl8n) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: slightly less displeased but it still has spaces 💢

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2019-05-25-spring-2019-fanart-contest-voting-now-open.md]: i will acquiesce on this one but i am of the firm opinion that em-dashes look like shit

**Pull request merged**